docs/Documentation/Schedules.md

```diff
@@ -0,0 +1,231 @@
+      notify_goodNight: 'notify &6Good night, sleep well!'
+    schedules:
+      sayGoodNight:
+        type: realtime-cron
+        time: '0 22 * * *'
+        events: bell_ring,notify_goodNight
+    ```
+
+    ---
+
+The following special expressions were added for extended functionality or simpler usage:
+
+| Expression              | Description                                                    | Equivalent to |
+|:------------------------|:---------------------------------------------------------------|:-------------:|
+| `@reboot`               | Run at server startup, before catching up any missed schedules |      _-_      |
+| `@hourly`               | Run once an hour at the beginning of the hour                  |  `0 * * * *`  |
+| `@daily` / `@midnight`  | Run once a day at 00:00                                        |  `0 0 * * *`  |
+| `@weekly`               | Run once a week at 00:00 on Sunday morning                     |  `0 0 * * 0`  |
+| `@monthly`              | Run once a month at 00:00 of the first day of the month        |  `0 0 1 * *`  |
+| `@yearly` / `@annually` | Run once a year at 00:00 of 1 January                          |  `0 0 1 1 *`  |
+
+### Catchup Strategies
+
+Obviously, scheduled events can't be run while the server is shut down.  
+If you want to be sure that a schedule will nevertheless be run, you can define a **catchup strategy**.
+
+On startup, BetonQuest checks which schedules have been missed and (if needed) they will be run on the first tick.  
+Schedules of the same type will be run in the order they were missed.
+For mixed types the **order** can **not** be **guaranteed**.
+
+=== "NONE"
+
+    ```YAML title="Example"
+    schedules:
+      sayGoodNight: 
+        type: realtime-daily
+        time: '22:00'
+        events: bell_ring,notify_goodNight
+        catchup: none #(1)!
+    ```
+
+    1.  Don't catch up any missed schedules after reboot.  
+        _(default value, can be ommitted)_
+
+    _As it's just an announcement we don't need to repeat it. The right time has passed._
+
+    ---
+
+
+=== "ONE"
+
+    ```YAML title="Example"
+    schedules:
+      resetQuarryArea: 
+        type: realtime-daily
+        time: '03:00'
+        events: pasteQuarry
+        catchup: one #(1)!
+    ```
+
+    1.  If the schedule was missed (no matter how often) run it **once** after reboot.
+
+    _The quarry should be reset every day at 3am. Even if the server was shut down at that time, run the event once at reboot._
+
+    ---
+
+
+=== "ALL"
+
+    ```YAML title="Example"
+    schedules:
+      greedyMarchant: 
+        type: realtime-daily
+        time: '03:00'
+        events: increaseFuelPrice
+        catchup: all #(1)!
+    ```
+
+    1.  Run the schedule as often as it was missed after reboot.
+
+    _Inflation! The fuel marchant increases it's price every day at 3am. If the sever was shut down, prices should increase at the same pace._
+
+    ---
```

Comment:
I think it should also work if you only define it once after the tabs, and not for every single tab